### PR TITLE
Strengthen configuration change approval

### PR DESCRIPTION
**Pull request by wuhuizuo**
Ref https://github.com/pingcap/tiflash/issues/8966

### What is changed and how it works?

Issue Number: Close #xxx

What's Changed:

```commit-message

```

### Related changes

- [ ] Need to cherry-pick to the release branch

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

### Release note

```release-note
None
```

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Sort the member alphabetically.
+aliases:
+  sig-critical-approvers-config:
+    - easonn7
+    - zhangjinpeng87
```
Comment (wuhuizuo) [inline OWNERS_ALIASES:5]:
```suggestion
    - yudongusa
```

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Sort the member alphabetically.
+aliases:
+  sig-critical-approvers-config:
+    - easonn7
+    - yudongusa
+  sig-approvers: # default repo approvers
+    - CalvinNeo
```
Comment (wuhuizuo) [inline OWNERS_ALIASES:7]:
```suggestion
    - CalvinNeo
    - zanmato1984
```

**Review by zanmato1984 — APPROVED** [review]

**Comment by wuhuizuo** [comment]
I do not known why the `build-check` CI task failed.

**Comment by CalvinNeo** [comment]
@wuhuizuo If we merged tikv/master, which introduces some new configurations, do we also need to approve it here?

**Comment by CalvinNeo** [comment]
> I do not known why the `build-check` CI task failed.

Because the free space provided by github is not enough

**Comment by wuhuizuo** [comment]
> @wuhuizuo If we merged tikv/master, which introduces some new configurations, do we also need to approve them here?

@CalvinNeo Sorry for the late reply.

No, here we just controlled the config.rs in `proxy_components/proxy_server/src` folder, unless we also merged the OWNERS of the upstream repository, if so it will need the approvals from the `tikv` approvers.

Of course, there are additional benefits at this time: knowing important changes and understanding the related impacts with experts. The focus at this time is to understand the upstream changes rather than the approval process (which happens less frequently).

**Comment by wuhuizuo** [comment]
/hold

it should be merged by @wuhuizuo

**Comment by wuhuizuo** [comment]
Ref: https://github.com/ti-community-infra/configs/pull/1067

**Review by CalvinNeo — APPROVED** [review]

**Pull request merged**